opennav_docking/include/opennav_docking/utils.hpp

```diff
@@ -141,6 +145,14 @@ inline bool parseDockParams(
     curr_dock.pose.position.y = pose_arr[1];
     curr_dock.pose.orientation = orientationAroundZAxis(pose_arr[2]);
 
+    if (!node->has_parameter(dock_name + ".id")) {
+      node->declare_parameter(dock_name + ".id", "");
+    }
+    if (!node->get_parameter(dock_name + ".id", curr_dock.id)) {
+      RCLCPP_ERROR(node->get_logger(), "Dock %s has no valid 'id'.", dock_name.c_str());
```

Comment:
Remove the error/return false, its OK if its not specified as an optional field. Otherwise will just be empty string.